proprietary/design-tokens/src/component/utrecht/link-social.tokens.json

```diff
@@ -0,0 +1,15 @@
+{
+  "utrecht": {
+    "link-social": {
+      "background-color": { "value": "{utrecht.color.red.40}" },
+      "border-width": { "value": "2px" },
+      "border-color": { "value": "{utrecht.color.white}" },
+      "size": { "value": "40px" },
+      "distanced": { "value": "9.6px" },
```

Comment:
Now that `0.6em` is converted to pixels, it is apparent that `9.6px` doesn't seem like a sensible value many high-dpi screens can render. I suggest we switch to `utrecht.space.inline.xs` or `utrecht.space.inline.sm` instead.

I would also suggest to rename `distanced` to `margin-inline-start` to match how other components work.